### ISSUE TITLE
How get pred prob of image mask with mmcv2.1.0 and mmseg 1.2.2?

### ISSUE THREAD

**Issue by funny000** [open-mmlab/mmsegmentation#3773]
Thank author for this work.
I want to get predict probability of image mask, and look the before issue find a same issue, but I use different version of  mmcv and mmseg, so don't know to change code to get predict probability, Can you tell me to how change code or use other function to get probability?
my use mmcv version is 2.1.0, and mmseg version is 1.2.2, mmegine version is 0.10.3.

**Comment by funny000** [comment]
I has look the issues, 
https://github.com/open-mmlab/mmsegmentation/issues/3528,
so can i  get the probability from "logits"?
